.github/workflows/pipeline.yml

```diff
@@ -52,6 +52,8 @@ jobs:
           java-version: ${{ matrix.java }}
           cache: 'maven'
       - name: Compile Java ${{ matrix.java }}
+        env:
```

Comment:
Please remove.

Reply:
I got a stack overflow exception during compiling with JDK 11 on the pipeline and increasing the stack size like this fixes it.

Reply:
The pipeline problem should have been fixed in #874. Please remove this change and merge the latest master branch into your code - the pipeline issue should go away.

Reply:
Yes, that fixes it. removed.